src/containers/shared/components/Transaction/OracleDelete/Simple.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { OracleDocumentID } = data.instructions
+  return (
+    <div className="oracle-delete">
+      {
+        // OracleDocumentID could be zero.
```

Comment:
This can be a TODO for a future PR, but do you think it'd be useful to have info here about the oracle that's being deleted?

Reply:
People seem to ask about way we could filter tx by `OracleDocumentID` so I deemed that showing the ID might be important